fxa-auth-db-server/docs/API.md

```diff
@@ -69,13 +69,13 @@ The following datatypes are used throughout this document:
     * deleteSessionToken        : `DEL /sessionToken/:id`
     * createSessionToken        : `PUT /sessionToken/:id`
     * updateSessionToken        : `POST /sessionToken/:id/update`
-    * verifyToken               : `POST /token/:tokenVerificationId/verify`
```

Comment:
It is used by @vbudhram's [sign-in confirmation PR](https://github.com/mozilla/fxa-auth-server/pull/1275/files#diff-ab87c506e682e952973178a41792df12R1140), which hasn't landed yet. So as long as this change is deployed ahead of that one, there should be nothing that depends on the old path.